eng/release/DiagnosticsReleaseTool/Core/Release.cs

```diff
@@ -224,14 +224,6 @@ private async Task<int> LayoutFilesAsync(CancellationToken ct)
 
                     if (layoutResult.Status == LayoutResultStatus.FileHandled)
                     {
-                        if (isProcessed)
-                        {
-                            // TODO: Might be worth to relax this limitation. It just needs to turn the
-                            //      source -> fileData relationship to something like source -> List<FileData>).
-                            _logger.LogError("[{buildFilePath}, {worker}] File {file} is getting handled by several workers.", file.FullName, worker.GetType().FullName, file);
-                            return -1;
-                        }
-
                         isProcessed = true;
```

Comment:
Do we need `isProcessed` anymore? It doesn't appear that it's currently being evaluated anywhere

Reply:
It's still used below to make sure that the file was actually processed and logs a warning if it was not.

Reply:
Ah, I see - another case of GitHub not expanding to show enough lines 👍